lsp-dart.el

```diff
@@ -114,8 +114,7 @@ If unspecified, diagnostics will not be generated."
   "Generate LSP startup command."
   (or lsp-dart-server-command
       (list (lsp-dart-dart-command)
-            (expand-file-name (f-join (lsp-dart-get-sdk-dir) "bin/snapshots/analysis_server.dart.snapshot"))
-            "--lsp"
+            "language-server"
```

Comment:
We need to take care with this since probably this flag was added in a specific dart SDK version, so I think the best would be to check dart-version and use this one after the version was included, also, I suggest checking how Dart-Code does that

Reply:
Ah good point. 

I'll double check that now